app/sim_test.go

```diff
@@ -104,20 +104,6 @@ func SetupSimulation(dirPrefix, dbName string) (simtypes.Config, dbm.DB, string,
 			}
 		}
 	}
-	// set sim params to overwrite wasmd default reflect.wasm path
-	simParams := simtypes.AppParams{
-		wasmsim.OpReflectContractPath: json.RawMessage(`"../testutil/wasm/reflect.wasm"`),
-	}
-
-	bz, err := json.Marshal(simParams)
-	if err != nil {
-		return simtypes.Config{}, nil, "", nil, false, sdkerrors.Wrap(err, "marshal sim custom params")
-	}
-	config.ParamsFile = filepath.Join(dir, "sim-params.json")
-	err = ioutil.WriteFile(config.ParamsFile, bz, 0o600)
-	if err != nil {
-		return simtypes.Config{}, nil, "", nil, false, sdkerrors.Wrap(err, "write temp sim params")
-	}
```

Comment:
No idea why it was needed. The default path should work properly, and the default one does not require to track the test contract `../testutil/wasm/reflect.wasm` version. So I delete it.